api_docs/changelog.md

```diff
@@ -22,16 +22,17 @@ format used by the Zulip server that they are interacting with.
 
 **Feature level 261**
 
-* [`POST /invites`](/api/send-invites), [`POST
-  /invites/multiuse`](/api/create-invite-link): Newly created user is
-  now subscribed to the default streams in the organization even when
-  the user sending the invite is not allowed to subscribe others.
```

Comment:
Combined this idea in the other bullet point, since this possibility really is connected to the parameter we added. Users who cannot subscribe other users to streams still need to pass an empty list to `stream_ids` for both endpoints.

Reply:
Yeah, this looks great.